core/src/commonTest/kotlin/com/sourcepoint/mobile_core/network/SourcepointClientTest.kt

```diff
@@ -369,69 +365,13 @@ class SourcepointClientTest {
         assertTrue(response.gdpr?.acceptedCategories?.isEmpty() == true)
     }
 
-    @Test
-    fun postGDPRChoiceActionContainCorrectUrl() = runTest {
```

Comment:
@andresilveirah, this test was moved here from iOS SDK as it is the most appropriate place here. We don't have test coverage of this code anywhere else, but here